React/Views/RCTTabBar.h

```diff
@@ -9,11 +9,22 @@
 
 #import <UIKit/UIKit.h>
 
+#import "RCTComponent.h"
+
+
 @interface RCTTabBar : UIView
 
 @property (nonatomic, strong) UIColor *unselectedTintColor;
 @property (nonatomic, strong) UIColor *tintColor;
 @property (nonatomic, strong) UIColor *barTintColor;
 @property (nonatomic, assign) BOOL translucent;
 
+/**
```

Comment:
onFocus and onBlur are already used as event names in TextInput, I would like to keep unique names to avoid confusion.  In any case, I can remove these for now, these should really be part of a later commit.